packages/transformers/webext/src/runtime/autoreload.js

```diff
@@ -0,0 +1,6 @@
+/* global window */
+window.addEventListener('beforeunload', function() {
```

Comment:
Would `module.hot.dispose` work here?

Reply:
If we did that, the module would reload on every hot reload, which is not only expensive but also means the web extension itself won't have true HMR. I've tested this locally and this is the best solution I found.